infra/index.ts

```diff
@@ -6,6 +6,7 @@ import "./src/dns";
 export * from "./src/metrics";
 export { dbConnectionString } from "./src/database";
 export { deployAccessKeyId, deployAccessKeySecret } from "./src/deployUser";
+export * from "./src/vanta";
```

Comment:
For whose benefit is this being exported?

Reply:
That was just so I could see the role ARN to copy into the Vanta UI. I could have gotten that from the AWS web dashboard as well though I guess.